server/store/db.js

```diff
@@ -0,0 +1,62 @@
+var mongoose = require('mongoose'),
+    config = require('../config/config.json').mongo;
+
+var options = {
+  server: {
```

Comment:
maybe it is a good idea to put those settings into `config.json` file instead of `db.js`. So it will be a configuration which does not require JS understanding and diving into the code